### PR TITLE
tools: harmonize download links

### PR DESCRIPTION
**Pull request by yikeke**
Via: https://github.com/pingcap/docs/pull/919/ PTAL @lilin90 @GregoryIan

### CODE REVIEW

File: tools/loader.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ category: tools
 
 Loader 是由 PingCAP 开发的数据导入工具，用于向 TiDB 中导入数据。
 
-[Binary 下载](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz)
+Loader 属于 TiDB 企业版工具集，请[在此下载](../tools/download.md)。
```
Comment (lilin90) [inline tools/loader.md]:
```suggestion
Loader 包含在 tidb-enterprise-tools 安装包中，可[点此下载](../tools/download.md)。
```

File: tools/mydumper.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ category: tools
 
 `mydumper` 是 [mydumper](https://github.com/maxbube/mydumper) 的 fork 项目，并添加了一些针对 TiDB 的功能。推荐使用此工具对 TiDB 进行逻辑备份。
 
-[下载 Binary](http://download.pingcap.org/tidb-enterprise-tools-latest-linux-amd64.tar.gz)。
+mydumper 属于 TiDB 企业版工具集，请[在此下载](../tools/download.md)。
```
Comment (lilin90) [inline tools/mydumper.md]:
```suggestion
mydumper 包含在 tidb-enterprise-tools 安装包中，可[点此下载](../tools/download.md)。
```

File: tools/syncer.md [hunk 3]
```diff
@@ -9,27 +9,12 @@ category: tools
 
 Syncer 是一个数据导入工具，能方便地将 MySQL 的数据增量导入到 TiDB。
 
-Syncer 属于 TiDB 企业版工具集，如何获取可参考[下载 TiDB 企业版工具集](#下载-tidb-企业版工具集-linux)。
+Syncer 属于 TiDB 企业版工具集，请[在此下载](../tools/download.md)。
```
Comment (lilin90) [inline tools/syncer.md]:
```suggestion
Syncer 包含在 tidb-enterprise-tools 安装包中，可[点此下载](../tools/download.md)。
```

File: tools/tidb-binlog.md [hunk 4]
```diff
@@ -32,22 +32,9 @@ Drainer 从各个 Pump 节点收集 Binlog，并按照在 TiDB 中事务的提
...
-    wget http://download.pingcap.org/tidb-binlog-local-linux-amd64.sha256
-
-    # 检查文件完整性，返回 ok 则正确
-    sha256sum -c tidb-binlog-local-linux-amd64.sha256
-
-    # 解开压缩包
-    tar -xzf tidb-binlog-local-linux-amd64.tar.gz
-    cd tidb-binlog-local-linux-amd64
-    ```
+TiDB-Binlog 属于 TiDB 企业版工具集，请[在此下载](../tools/download.md)。
```
Comment (lilin90) [inline tools/tidb-binlog.md]:
```suggestion
TiDB-Binlog 包含在 tidb-enterprise-tools 安装包中，可[点此下载](../tools/download.md)。
```

Comment (yikeke) [reply tools/tidb-binlog.md]: Updated. PTAL. @lilin90

**Pull request merged**